ILRepack/RepackImporter.cs

```diff
@@ -119,7 +119,11 @@ public MethodReference Import(MethodReference reference, IGenericParameterProvid
 
         public TypeDefinition Import(TypeDefinition type, Collection<TypeDefinition> col, bool internalize)
         {
-            _logger.Verbose("- Importing " + type);
+			if (CanInclude(type) == false) {
+				_logger.Log("Repack dropped " + type.FullName);
```

Comment:
This is also logged in the `CanInclude` method, I'd say we only log once here, and rewrite it as; `Type <X> dropped as it was marked with the drop attribute`

Reply:
You are right. I forgot that CanInclude also logged that. Shall I simply remove the log here?